context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+This programming language seems to be primarily for graphic designers (either professionally or casually) who have some experience writing code and thinking mathematically. I could also see mathematicians who focus on geometry using it to play around with an idea, since it easily represents fractal images that would be hard to represent manually with something like TIKz.
```

Comment:
I like the point about mathematicians, hadn't thought of that. I guess some recursive/fractal style visuals would be easier here than elsewhere.